src/ipahealthcheck/ipa/trust.py

```diff
@@ -684,3 +684,22 @@ def check(self):
                          key='adtrustpackage',
                          msg='trust-ad sub-package is not installed. '
                          'Administration will be limited.')
+
+
+@registry
+class IPAauthzdatapacCheck(IPAPlugin):
+    """
+    Verify that the MS-PAC generation is not disabled
+    """
+    @duration
+    def check(self):
+        ipaconfig = api.Command.config_show(raw=True)
+        krbauthzdata = ipaconfig['result'].get('ipakrbauthzdata', tuple())
+        for authzdata in ('MS-PAC', ):
```

Comment:
I see what you are going for here, making this more future-proof, but the rest of the method is not generic at all. I think we're ok simplifying this to simply check for MS-PAC and if we need to extend it later we can do that.
The way it is now has the side-effect of returning both ERROR and SUCCESS if MS-PAC is not enabled.

Reply:
Fun thing: I did the change you wanted (and added in the proposal below) but somehow forgot to amend the commit with that before opening the PR.